src/js/stores/IntercomStore.js

```diff
@@ -0,0 +1,41 @@
+import PluginSDK from "PluginSDK";
+
+import GetSetBaseStore from "./GetSetBaseStore";
+import { INTERCOM_CHANGE } from "../constants/EventTypes";
+
+class IntercomStore extends GetSetBaseStore {
+  constructor() {
+    super(...arguments);
+
+    this.getSet_data = {
```

Comment:
This seems unused, why do we need it?

Reply:
Please look inside `GetSetBaseStore` this is the way to define internal scope `attributes` so that you could do `this.get("attributes")`